.gitsplit.yml

```diff
@@ -20,6 +20,22 @@ splits:
     target: "https://${GH_TOKEN}@github.com/opentelemetry-php/sdk.git"
   - prefix: "src/Contrib"
     target: "https://${GH_TOKEN}@github.com/opentelemetry-php/sdk-contrib.git"
+  - prefix: "src/Contrib/Jaeger"
```

Comment:
With the recent PR merged, there is an addition for `src/Extension` in .gitsplit.yml.